lib/fathead/beautifulsoup/output.txt

```diff
@@ -1,69 +1,69 @@
+find_all_next and find_next	A							https://www.crummy.com/software/BeautifulSoup/bs4/doc/			<section class="prog__container"><pre><code>Signature: find_all_next(name, attrs, string, limit, **kwargs)</pre></code><pre><code>Signature: find_next(name, attrs, string, **kwargs)</pre></code><p>These methods use .next_elements to iterate over whatever tags and strings that come after it in the document. The find_all_next() method returns all matches, and find_next() only returns the first match:</p><pre><code>first_link = soup.a\nfirst_link\n# <a class="sister" href="http://example.com/elsie" id="link1">Elsie</a>\n\nfirst_link.find_all_next(string=True)\n# [u'Elsie', u',\n', u'Lacie', u' and\n', u'Tillie',\n#  u';\nand they lived at the bottom of a well.', u'\n\n', u'...', u'\n']\n\nfirst_link.find_next("p")\n# <p class="story">...</p>\n\n</code></pre></section>	https://www.crummy.com/software/BeautifulSoup/bs4/doc/#find-all-next-and-find-next
+find_all_previous and find_previous	A							https://www.crummy.com/software/BeautifulSoup/bs4/doc/			<section class="prog__container"><pre><code>Signature: find_all_previous(name, attrs, string, limit, **kwargs)</pre></code><pre><code>Signature: find_previous(name, attrs, string, **kwargs)</pre></code><p>These methods use .previous_elements to iterate over the tags and strings that came before it in the document. The find_all_previous() method returns all matches, and find_previous() only returns the first match:</p><pre><code>first_link = soup.a\nfirst_link\n# <a class="sister" href="http://example.com/elsie" id="link1">Elsie</a>\n\nfirst_link.find_all_previous("p")\n# [<p class="story">Once upon a time there were three little sisters; ...</p>,\n#  <p class="title"><b>The Dormouse's story</b></p>]\n\nfirst_link.find_previous("title")\n# <title>The Dormouse's story</title>\n\n</code></pre></section>	https://www.crummy.com/software/BeautifulSoup/bs4/doc/#find-all-previous-and-find-previous
+css selectors	A							https://www.crummy.com/software/BeautifulSoup/bs4/doc/			<section class="prog__container"><p>Beautiful Soup supports the most commonly-used CSS selectors. Just pass a string into the .select() method of a Tag object or the BeautifulSoup object itself.</p><pre><code>soup.select("title")\n# [<title>The Dormouse's story</title>]
```

Comment:
@vidhan13j07 it looks like this like and several others contain `\x` which is causing problems for our parser. Can you please escape them in the parsing script?

Something like `s/\\x/\\\\x/` should work.